.gitignore

```diff
@@ -135,3 +135,8 @@ dmypy.json
 /.idea/inspectionProfiles/profiles_settings.xml
 /.idea/inspectionProfiles/Project_Default.xml
 /.idea/vcs.xml
+
+# User
+/001_baseball_pitching_right_handed.trc
+/recommendation_metrics.csv
+.env*
```

Comment:
i was thinking of having a file called `.env_example` actually committed to the repo that would demo what this file should look like, so maybe remove the * here and just ignore the .env file specifically?